versions/1/spec.yml

```diff
@@ -28,6 +28,17 @@ spec:
     type: folder
     name: docs
     required: true
+  - description: Folder containing agent-related definitions
+    type: folder
+    name: agent
+    required: false
+    additionalContents: false
+    contents:
+    - description: Folder containing input definitions
+      type: folder
+      name: input
+      required: true
+      additionalContents: false
```

Comment:
Related to @ruflin's comment, it would be good to flesh out `contents` here with a spec for the file(s) that could be be contained in this folder.